extensions/azurecore/src/azureResource/services/subscriptionFilterService.ts

```diff
@@ -54,19 +54,21 @@ export class AzureResourceSubscriptionFilterService implements IAzureResourceSub
 			filters.push(...selectedSubscriptionsCache[accountId].map((subcription) => `${accountId}/${subcription.id}/${subcription.name}`));
 		}
 
-		const resourceFilterConfig = this._config.inspect<string[]>(AzureResourceSubscriptionFilterService.filterConfigName);
-		let configTarget = ConfigurationTarget.Global;
-		if (resourceFilterConfig) {
-			if (resourceFilterConfig.workspaceFolderValue) {
-				configTarget = ConfigurationTarget.WorkspaceFolder;
-			} else if (resourceFilterConfig.workspaceValue) {
-				configTarget = ConfigurationTarget.Workspace;
-			} else if (resourceFilterConfig.globalValue) {
-				configTarget = ConfigurationTarget.Global;
+		if (this._config) {
```

Comment:
This part of the code has essentially always been broken. We just didn't notice it due to stuff eating our errors.